### PR TITLE
Ads 1047

### PR DESCRIPTION
**Pull request by andrewgeorgiou1981**
@adgad @VladDubrovskis @gvonkoss

### CODE REVIEW

File: views/article-list.html [hunk 1]
```diff
@@ -93,6 +95,26 @@
 				{{#if hasStreamListTitle}} data-has-stream-list-title{{/if}}
 			>
 				<div class="o-grid-row">
+					{{#if ads.twoMidRailAds}}
+
+						{{#unless properties.streamListTrendingTopics}}
+							{{#with trendingTopics}}
```
Comment (adgad) [inline views/article-list.html]: How does this work? If you're in an unless streamListTrendingTopics wouldn't that mean there would be no trendingTopics property?

File: views/article-list.html [hunk 2]
```diff
@@ -93,6 +95,26 @@
...
+									{{{reactRenderToString ../components.Section spread=@this}}}
+								</div>
+							{{/with}}
+						{{/unless}}
+
+						{{#if topSection}}
+							{{>ads/mid1-both-large}}
+						{{else}}
+							{{>ads/mid-both-large}}
+							<br><br><br>
```
Comment (adgad) [inline views/article-list.html]: What's this for?

File: views/partials/ads/mid1-both-large.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div
+	class="o-ads advert--mid1-both-large"
+	data-o-ads-name="mid1-both-large"
+	data-o-ads-targeting="pos=mid1;"
+	data-o-ads-center="true"
+	data-o-ads-label="false"
+	data-o-ads-formats-default="MediumRectangle,HalfPage,Responsive"
```
Comment (adgad) [inline views/partials/ads/mid1-both-large.html]: I'm pretty sure default isn't a thing anymore, maybe we could delete it from these templates as it looks a little odd?

File: views/article-list.html (hunk 2 above)
Comment (andrewgeorgiou1981) [inline views/article-list.html]: removed this.

File: views/article-list.html (hunk 1 above)
Comment (andrewgeorgiou1981) [inline views/article-list.html]: Correct - this was a bug with my code, i've corrected it. good spot.

**Pull request merged**